smithy-aws-cloudformation/src/main/java/software/amazon/smithy/aws/cloudformation/schema/fromsmithy/CfnConverter.java

```diff
@@ -185,8 +193,7 @@ private List<ConversionEnvironment> createConversionEnvironments(Model model) {
     private ConversionEnvironment createConversionEnvironment(
             Model model,
             ServiceShape serviceShape,
-            ResourceShape resourceShape
-    ) {
```

Comment:
Nit: this is the formatting typically used elsewhere

Reply:
Adjusted the formatting so the close parens and open brace are on their own line, not indented.